daml/Demo.daml

```diff
@@ -27,6 +27,9 @@ demo = do
   bobUsd      <- depositAsset providers bob       (usd with quantity = 1_000_000.0) bob.mainAccount.id
```

Comment:
I have a separate account here since the clearinghouse will need to be an observer on the on the `AssetSettlementRule`, and I don't think the customer will always want the clearinghouse to be an observer on their unrelated deposits. It would be fine to use the `mainAccount` for testing w/ the script though.

This does bring up an issue when onboarding customers not via the script - I could have the `Offer`/`Request` choices add the clearinghouse as a controller on the `AssetSettlementRule` for the given clearing account (which would add the as an observer as well as remove the need to do `transferWithController` in the `TransferFromProvider` choice), but any deposits in the account beforehand would still be missing the clearinghouse as an observer. It might need to be a fresh account.

Any ideas for this?

Reply:
@georg-da @brian-weir ?

Reply:
Good question - I believe the answer lies with what privacy the account should have as. The `mainAccount` has the **exchange** party as an observer which allows for the settlement of transactions based off executions by the **exchange**. Also adding the **clearinghouse** for demo purposes is fine but like the **exchange** party, I'm not sure we'd want these parties to be able to see all the `Transfer` and `Credit` events associated to an account via this `AssetSettlementRule`.